consul/src/main/java/com/linecorp/armeria/internal/consul/Check.java

```diff
@@ -0,0 +1,493 @@
+/*
+ * Copyright 2019 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+package com.linecorp.armeria.internal.consul;
+
+import java.util.Map;
+
+import javax.annotation.Nullable;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonInclude.Include;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.google.common.base.MoreObjects;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+@JsonInclude(Include.NON_NULL)
+public final class Check {
+
+    @Nullable
+    @JsonProperty("Node")
+    private String node;
+
+    @Nullable
+    @JsonProperty("CheckID")
+    private String checkId;
+
+    @Nullable
+    @JsonProperty("Name")
+    private String name;
+
+    @Nullable
+    @JsonProperty("ID")
+    private String id;
+
+    @Nullable
+    @JsonProperty("Interval")
+    private String interval = "10s";
+
+    @Nullable
+    @JsonProperty("Notes")
+    private String notes;
+
+    @Nullable
+    @JsonProperty("DeregisterCriticalServiceAfter")
+    private String deregisterCriticalServiceAfter;
+
+    @Nullable
+    @JsonProperty("Args")
+    private String[] args;
+
+    @Nullable
+    @JsonProperty("AliasNode")
+    private String[] aliasNode;
+
+    @Nullable
+    @JsonProperty("DockerContainerID")
+    private String dockerContainerID;
+
+    @Nullable
+    @JsonProperty("GRPC")
+    private String grpc;
+
+    @JsonProperty("GRPCUseTLS")
+    private boolean grpcUseTls;
+
+    @Nullable
+    @JsonProperty("Shell")
+    private String shell;
+
+    @Nullable
+    @JsonProperty("HTTP")
+    private String http;
+
+    @Nullable
+    @JsonProperty("Method")
+    private String method;
+
+    @Nullable
+    @JsonProperty("Header")
+    private Map<String, String[]> header;
+
+    @Nullable
+    @JsonProperty("Timeout")
+    private String timeout = "10s";
+
+    @JsonProperty("OutputMaxSize")
+    private int outputMaxSize = 4096;
+
+    @JsonProperty("TLSSkipVerify")
+    private boolean tlsSkipVerify;
+
+    @Nullable
+    @JsonProperty("TCP")
+    private String tcp;
+
+    @Nullable
+    @JsonProperty("TTL")
+    private String ttl;
+
+    @Nullable
+    @JsonProperty("ServiceID")
+    private String serviceId;
+
+    @Nullable
+    @JsonProperty("Status")
+    private String status;
+
+    /**
+     * Getter for node.
```

Comment:
This is internal class. We could remove Javadoc for getters and setters.

Reply:
Yes.